src/pages/Contact.js

```diff
@@ -7,10 +7,11 @@ function Contact() {
   const { t, i18n } = useTranslation();
 
   return (
-    <GridContainer containerSize="widescreen">
+    <GridContainer containerSize="widescreen" id="main-content">
```

Comment:
I notice that sometimes you use className and sometimes you use id. It might be easier to keep everything consistent

Reply:
Good suggestion! I'm using the ID's as anchor tags for my "skip to main content" link and another one for my "navigate to top" link, so I'm not sure how I'd use a className for this.